README.md

```diff
@@ -198,6 +198,7 @@ worst, "hang" (never terminate).
 | `armv7-unknown-linux-gnueabihf`      | 2.15   | 4.6.2   | 1.0.2m  | ✓   | 2.8.0 |   ✓    |
 | `armv7-unknown-linux-musleabihf`     | 1.1.15 | 5.3.1   | N/A     |     | 2.8.0 |   ✓    |
 | `asmjs-unknown-emscripten` [4]       | 1.1.15 | 1.37.13 | N/A     | ✓   | N/A   |   ✓    |
+| `i586-unknown-linux-gnu`             | 2.15   | 4.6.2   | 1.0.2m  | ✓   | N/A   |   ✓    |
```

Comment:
the glibc version is wrong. Ubuntu 17.04 ships with glibc 2.24.

Reply:
I've downgraded the ubuntu version to 12.04, so that this should now be correct.